### ISSUE TITLE
Hello, I want to know the difference between PBNS_Resizer and PBNS

### ISSUE THREAD

**Issue by wtupdl** [HugoBA92/PBNS#14]
Hello, I want to know the difference between PBNS_Resizer and PBNS

**Comment by hbertiche** [comment]
Hi,

thanks for your interest in my work!

PBNS is a solution for the outfit animation problem,
within this scope, this is defined as the task of deforming the outfit geometry according to the body pose,

on the other hand, PBNS_Resizer, while it works similarly, it _animates_ the outfit based on the body shape, not the pose,
the outfit geometry will change to adapt to the body shape changes (see SMPL model to understand body shape changes),
additionally, PBNS_Resizer allows also for two additional parameters to represent tightness, as to control the size of the garment without modifying the body,

TL;DR: PBNS animates outfits as a function of the body pose. PBNS_Resizer animates outfits as a function of the body shape.

I hope this helps,
please do not hesitate to ask further questions
